src/components/Pursuance/PursuancePage.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import PursuanceMenu from './PursuanceMenu';
+import TaskListView from './views/TaskListView';
+import './PursuancePage.css';
+
+class PursuancePage extends Component {
+
+  render() {
+    return (
+      <Router>
+        <div className="PursuancePage">
+          <nav>
+            <PursuanceMenu
+              pursuanceId={this.props.match.params.pursuanceId} />
+          </nav>
+          <article>
+            <Switch>
+              <Route exact path="/pursuance/:pursuanceId" component={TaskListView} />
```

Comment:
@Spetastian I'm glad you added the greenification of menu icons to show people which page they're on!  Unfortunately this route, there is no such greenifying of the `Task` icon.  Could you redirect from `/pursuance/:pursuanceId` to `/pursuance/:pursuanceId/tasks` please?

Reply:
Fixed!